.github/workflows/InitDev Postgres, Smoke Tests.yml

```diff
@@ -26,7 +26,7 @@ env:
   BUILD_INCREMENTER: "1"
   CONFIGURATION: "Release"
   VERSION_MAJOR: "6"
-  VERSION_MINOR: "1"
+  VERSION_MINOR: "2"
   HEAD_REF: ${{ GITHUB.HEAD_REF }}
   REF_NAME: ${{ GITHUB.REF_NAME }}
   REPOSITORY_DISPATCH_BRANCH: ${{ github.event.client_payload.branch }}
```

Comment:
Below line 57, let's add `ref: ${{ github.head_ref }}` so that we checkout the correct branch. Right now this workflow is checking out `main` for `Ed-Fi-ODS`. Same change needs to be made in the other workflow files in this repository.